### PR TITLE
some GalaxyManager cleanup

### PR DESCRIPTION
**Pull request by Punikekk**

### CODE REVIEW

File: src/main/java/com/github/manolo8/darkbot/backpage/entities/galaxy/GalaxyInfo.java [hunk 1]
```diff
@@ -1,11 +1,12 @@
 package com.github.manolo8.darkbot.backpage.entities.galaxy;
 
-import com.github.manolo8.darkbot.utils.XmlHelper;
 import org.dom4j.Element;
 
 import java.util.List;
 import java.util.stream.Collectors;
 
+import static com.github.manolo8.darkbot.utils.XmlHelper.*;
```
Comment (Pablete1234) [inline src/main/java/com/github/manolo8/darkbot/backpage/entities/galaxy/GalaxyInfo.java:8]: avoid static imports, and avoid wildcard imports

File: src/main/java/com/github/manolo8/darkbot/backpage/entities/galaxy/Gate.java [hunk 2]
```diff
@@ -1,8 +1,9 @@
 package com.github.manolo8.darkbot.backpage.entities.galaxy;
 
-import com.github.manolo8.darkbot.utils.XmlHelper;
 import org.dom4j.Element;
 
+import static com.github.manolo8.darkbot.utils.XmlHelper.getAttrInt;
```
Comment (Pablete1234) [inline src/main/java/com/github/manolo8/darkbot/backpage/entities/galaxy/Gate.java:5]: same here, just avoid the static import

File: src/main/java/com/github/manolo8/darkbot/backpage/entities/galaxy/Item.java [hunk 3]
```diff
@@ -1,8 +1,9 @@
 package com.github.manolo8.darkbot.backpage.entities.galaxy;
 
-import com.github.manolo8.darkbot.utils.XmlHelper;
 import org.dom4j.Element;
 
+import static com.github.manolo8.darkbot.utils.XmlHelper.getAttrInt;
```
Comment (Pablete1234) [inline src/main/java/com/github/manolo8/darkbot/backpage/entities/galaxy/Item.java:5]: same here, avoid static import

File: src/main/java/com/github/manolo8/darkbot/backpage/entities/galaxy/Multiplier.java [hunk 4]
```diff
@@ -1,21 +1,22 @@
 package com.github.manolo8.darkbot.backpage.entities.galaxy;
 
-import com.github.manolo8.darkbot.utils.XmlHelper;
 import org.dom4j.Element;
 
+import static com.github.manolo8.darkbot.utils.XmlHelper.getAttrInt;
```
Comment (Pablete1234) [inline src/main/java/com/github/manolo8/darkbot/backpage/entities/galaxy/Multiplier.java:5]: same here, avoid static import

**Review by Pablete1234 — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/manolo8/darkbot/backpage/entities/galaxy/GalaxyInfo.java (hunk 1 above)
Comment (Punikekk) [reply src/main/java/com/github/manolo8/darkbot/backpage/entities/galaxy/GalaxyInfo.java:8]: why 🤔

Comment (Pablete1234) [reply src/main/java/com/github/manolo8/darkbot/backpage/entities/galaxy/GalaxyInfo.java:8]: For static imports, it's really not intuitive to see a function call without knowing it comes from another file

**Pull request closed without merging**